roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile: dest=/etc/yum/pluginconf.d/fastestmirror.conf regexp="^enabled=.*" line="enabled=0" state=present
```

Comment:
Please use long form syntax. Short form is less readable.
```
- name: Disable fastestmirror plugin
  lineinfile:
    dest: /etc/yum/pluginconf.d/fastestmirror.conf
    regexp: "^enabled=.*"
    line: "enabled=0"
    state: present
```
Secondly, you should add `create: yes` in case this file is missing.